var/spack/repos/builtin/packages/rocrand/package.py

```diff
@@ -120,13 +120,30 @@ class Rocrand(CMakePackage):
     # same directory.
     patch("hiprand_prefer_samedir_rocrand.patch", working_dir="hiprand", when="@5.2.0:")
 
+    # Add hiprand sources thru the below
+    for d_version, d_commit in [
+        ("5.3.0", "125d691d3bcc6de5f5d63cf5f5a993c636251208"),
+        ("5.2.3", "7190cc2199d8d97dbb6ce3fd179f1a62ad8fb13b"),
+        ("5.2.1", "7190cc2199d8d97dbb6ce3fd179f1a62ad8fb13b"),
+        ("5.2.0", "7190cc2199d8d97dbb6ce3fd179f1a62ad8fb13b"),
+        ("5.1.3", "7190cc2199d8d97dbb6ce3fd179f1a62ad8fb13b"),
+        ("5.1.0", "7190cc2199d8d97dbb6ce3fd179f1a62ad8fb13b"),
+    ]:
+        resource(
+            name="hipRAND",
+            git="https://github.com/ROCmSoftwarePlatform/hipRAND.git",
+            commit=d_commit,
+            destination="",
+            placement="hiprand",
+            when="@{0}".format(d_version),
+        )
     resource(
         name="hipRAND",
         git="https://github.com/ROCmSoftwarePlatform/hipRAND.git",
         branch="develop",
         destination="",
         placement="hiprand",
-        when="@5.1.0:",
+        when="@master",
     )
```

Comment:
For which you'd need a second one of these with `when="@develop"`.

Reply:
Thank you..!